.github/workflows

```diff
@@ -0,0 +1,50 @@
+  FORCE_COLOR: 1
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-22.04
```

Comment:
> Set up [a linter for HTML and CSS](https://github.com/microverseinc/linters-config) and make sure that any files that are not relevant are ignored by git.

- [ ] I can see you tried setting up linters for your project, but I think the setup is not complete and therefore the linters are not running. The structure of the linters folder is. You create a `.github folder` then you create a `workflows folder` inside it, then you create a `linters.yml file` inside it. Try to keep in mind the difference between folders and files here. Here is the [link to the linters setup repository,](https://github.com/microverseinc/linters-config) Follow the direction and set up the linters correctly for your project. :wave: :+1: